core/keyserver/keyserver.go

```diff
@@ -132,7 +136,7 @@ func (s *Server) getEntry(ctx context.Context, userID, appID string, firstTreeSi
 			LogId: s.logID,
 		})
 	if err != nil {
-		log.Printf("tlog.GetLatestSignedLogRoot(%v): %v", s.logID, err)
+		glog.Warningf("tlog.GetLatestSignedLogRoot(%v): %v", s.logID, err)
```

Comment:
This one should have been a warning. 

In general I'm using errors for internal failures, and warnings for input validation coming from the user.